### PR TITLE
Added ADR to the repository

### PR DESCRIPTION
**Pull request by michaelkotelnikov**
Added the ADR to the repository.

### CODE REVIEW

**Comment by michaelkotelnikov** [comment]
Related to https://github.com/openshift-4-compliance/openshift-4-compliance-automation/issues/27

File: ADR.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+# Programming languages
```
Comment (igorkchyts) [inline ADR.md:1]: @michaelkotelnikov 
The title relation to the topic is unclear.
Suggest: Policies Engines for OpenShift

File: ADR.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  * [Implications](#implications)
+* [Related](#related)
+  * [Related decisions](#related-decisions)
+  * [Related requirements](#related-requirements)
+  * [Related artifacts](#related-artifacts)
+  * [Related principles](#related-principles)
+* [Notes](#notes)
+
+
+## Summary
```
Comment (igorkchyts) [inline ADR.md:23]: @michaelkotelnikov 
No reason providing empty spaces

**Review by igorkchyts — COMMENTED** [review]
Looks Great!

**Review by igorkchyts — APPROVED** [review]

**Pull request merged**